src/importexport/guitarpro/internal/importgtp.cpp

```diff
@@ -1332,6 +1332,34 @@ bool GuitarPro1::read(QFile* fp)
     return true;
 }
 
+int GuitarPro::harmonicOvertone(Note* note, float harmonicValue, int harmonicType)
+{
+    int result{ 0 };
+
+    if (harmonicValue == 12.0f) {
```

Comment:
Checking floating point numbers for equality is asking for trouble. Some sort of epsilon is needed or the corresponding method Qt provides for this: [qFuzzyCompare()](https://doc.qt.io/qt-5/qtglobal.html#qFuzzyCompare)

Reply:
Fixed. I have founded mu::RealIsEqual utilite in global module

Reply:
That probably does use qFuzzyCompare under the covers?

Reply:
mu::RealIsEqual use its own realisation. But it is the same actually.

Reply:
I see. Not sure though why reinventing the wheel there rather than taking an existing method of a library we use anyway.

Reply:
and qFuzzyCompare is used in a lot of other places in the source